ui.tests/test-module/package.json

```diff
@@ -1,53 +1,33 @@
 {
-  "name": "aem-cloud-wdio-sample-project",
+  "name": "cypress-eaas",
   "version": "1.0.0",
-  "description": "Sample Tests Project",
-  "author": "catalan@adobe.com",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/adobe/aem-project-archetype.git"
-  },
-  "license": "SEE LICENSE IN LICENSE file",
-  "keywords": [
-    "aem",
-    "cloud",
-    "aemaacs",
-    "selenium",
-    "wdio"
-  ],
+  "description": "test cypress on eaas",
```

Comment:
hm, rather rename the package name/description to include WKND in some way?

Reply:
https://github.com/adobe/aem-test-samples/blob/aem-cloud/ui-cypress/test-module/package.json#L4